libass/ass_rasterizer.c

```diff
@@ -258,13 +264,19 @@ static bool add_cubic(RasterizerData *rst, const OutlinePoint *pt)
 }
 
 
-bool rasterizer_set_outline(RasterizerData *rst, const ASS_Outline *path)
+bool rasterizer_set_outline(RasterizerData *rst,
+                            const ASS_Outline *path, bool extra)
 {
     enum Status {
         S_ON, S_Q, S_C1, S_C2
     };
 
-    rst->size[0] = 0;
+    if (!extra) {
+        rst->x_min = rst->y_min = 0x7FFFFFFF;
+        rst->x_max = rst->y_max = 0x80000000;
```

Comment:
Do you mean INT32_MIN/INT32_MAX?
Well, those lines is copy-paste but indeed, using compiler constants make sense.